automerge-backend/src/patch.rs

```diff
@@ -48,6 +68,17 @@ pub struct Diff {
     pub conflicts: Vec<Conflict>,
 }
 
+impl Diff {
+    pub fn links(&self) -> Vec<ObjectID> {
+        let mut oids = vec![];
+        self.action.value().map(|v| v.object_id().map(|oid| oids.push(oid)));
```

Comment:
Probably could use `Option::and_then` rather than nested `map` calls like this. Personally I dislike using `map` for side effecting things so probably would write this as:

```
if let Some(oid) = self.action.value().and_then(|v| v.object_id()) { oids.push oid }
```